.github/workflows/benchmarks-reusable.yml

```diff
@@ -146,7 +138,7 @@ jobs:
     - name: Configure UR
```

Comment:
not sure if this makes any difference, but above there's still  `-DSYCL_UR_SOURCE_DIR=${{github.workspace}}/ur-repo/"` (probably just omitted, but pls confirm)

Reply:
Done. Thanks!